sample.json

```diff
@@ -0,0 +1,13 @@
+{
+    "sender": "0x6B5f6558CB8B3C8Fec2DA0B1edA9b9d5C064ca47",
+    "nonce": "0xf",
+    "initCode": "0x",
+    "callData": "{\"chainId\":80001, \"sender\":\"0x0A7199a96fdf0252E09F76545c1eF2be3692F46b\",\"kind\":\"swap\",\"hash\":\"\",\"sellToken\":\"TokenA\",\"buyToken\":\"TokenB\",\"sellAmount\":10,\"buyAmount\":5,\"partiallyFillable\":false,\"status\":\"Received\",\"createdAt\":0,\"expirationAt\":0}",
```

Comment:
It would be nice to update the example with a realistic Intent json that uses real tokens, i.e. USDC.